Create CSS file

RoderickSang opened the pull request:
Website was missing CSS file. Adding what I wrote into it. #3

RoderickSang committed:
Create CSS file

Website was missing CSS file. Adding what I wrote into it.

mikefey commented on main.css:
```diff
@@ -0,0 +1,40 @@
+    padding-top: 0;
+    padding-bottom: 0;
+    line-height: 1em;
+}
```
This means that you haven't added a new line to the end of the file, which is not 100% necessary but is considered a best practice (https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file)

mikefey replied:
This little icon means that you haven't added a new line to the end of the file, which is not 100% necessary but is considered a best practice (https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file)

mikefey commented on main.css:
```diff
@@ -0,0 +1,40 @@
+    font-weight: 600;
+    font-size: 20px;
+    letter-spacing: 2px;
+    font-family: futura-pt;
```
Looks like there are 2 different `font-family` declarations here - I'm not sure which one you want to use, but the way it's currently written, the first one will be ignored, so only `futura-pt` will be used.

mikefey commented on main.css:
```diff
@@ -0,0 +1,40 @@
+.nav-links {
+    list-style: none;
+    font-family: "proxima-nova","Helvetica Neue",Helvetica,Arial,sans-serif; /*Figure out a new font*/
+
```
Very small thing but there is an unnecessary space here between `font-family` and `display` (unless that was intentional).

mikefey reviewed (approved):
Looks great! I just left a few comments.

RoderickSang committed:
Remove unnecessary space

RoderickSang committed:
Final Fixes

Removed the extra font-family that I didn't need and also added a blank line at the end.
